spec.emu

```diff
@@ -316,6 +327,7 @@ contributors: Google, Ecma International
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
       <li>[[Style]] is one of the String values `"narrow"`, `"short"`, or `"long"`, identifying the display names style used.</li>
       <li>[[Type]] is one of the String values `"language"`, `"region"`, `"script"`, `"currency"`, `"dateField"`, or `"dateSymbol"`, identifying the type of the display names requested.</li>
+      <li>[[Substitute]] is one of the String values `"fallback"`, or `"none"`, identifying what to return while the system does not have the requested display name.</li>
```

Comment:
Have you done research on precedent in the web platform on possible naming for this sort of feature?  The first thing that comes to mind is multiple load paths for modules.

The words "substitute" and "fallback" seem too redundant to me, but that might be my bias as an ICU developer.  Maybe something more specific like `substitute: "code"` or `substitute: "use-input"`, or use one of those keys with `fallback` as the key?

Reply:
how about substitute: 'fallbackToCode' , 'none'

Reply:
"fallback" is [a noun](https://www.merriam-webster.com/dictionary/fallback). When used as a verb, there are two words. Also, as mentioned in #29, we use kebab case for strings. So, the option you proposed would be, `substitute: "fall-back-to-code"`. That could work, but it is a bit wordy.

Reply:
I somehow completely missed 49f62f0afd79b367981a0f6624257f2293416d82 and have been operating from the perspective of `Intl.DisplayNames.prototype.of( codes )`. With scalar input, is there even sufficient value in making this configurable as opposed to always returning undefined? I would expect fallback to be implemented by user code like `codes.map(code => displayNameFormatter.of(code) || code)`.

But if we do want to keep it and are bikeshedding, I'd propose `fallback: "code"` and `fallback: "none"`.

Reply:
> With scalar input, is there even sufficient value in making this configurable as opposed to always returning undefined?

My positions recorded in the notes still stand in favor of the option.

> I'd propose fallback: "code" and fallback: "none".

LGTM